.travis.yml

```diff
@@ -44,7 +44,6 @@ env:
     - REQUIREMENTS=devel EXTRAS=all,postgresql,elasticsearch7 SQLALCHEMY_DATABASE_URI="postgresql+psycopg2://postgres@localhost:5432/invenio" ES_URL=$ES7_DOWNLOAD_URL DEPLOY=true
 
 python:
-  - "2.7"
```

Comment:
Can you add 3.5 (just because it's declared like that originally in v3.2 release).